rebasehelper/checkers/rpmdiff_tool.py

```diff
@@ -55,6 +57,51 @@ def get_checker_name(cls):
     def is_default(cls):
         return cls.DEFAULT
 
+    @classmethod
+    def get_license_changes(cls):
+        """
+        Finds out if there is a possibility of license change in new sources
+
+        :return: true if license changes probably happened
+        """
+        diffs = []
+        for srcs in ['new_sources', 'old_sources']:
+            srcs = os.path.join(cls.workspace_dir, srcs)
+
+            # Get all files
+            files = PathHelper.find_all_files(srcs, '*')
+
+            # Get changed files
+            changed = [f.split('/')[-1] for f in cls.results_dict['changed']]
+
+            diff = set()
+            for f in files:
+                # Check only changed files
+                if f.split('/')[-1] not in changed:
+                    continue
+
+                # Do not analyse .git directory files
+                if r'.git' in f:
+                    continue
+
+                out = subprocess.Popen(["/bin/licensecheck", f, "--machine"],
+                                       stdout=subprocess.PIPE)
+                # Get licensecheck output
+                out = out.stdout.read()
```

Comment:
Why don't you use `ProcerssHelper` for this?

Reply:
Also, any reason not to run `licensecheck` for all files at once?

Reply:
Also why `/bin`? `licensecheck` package owns `/usr/bin/licensecheck`, and in Fedora `/bin` is just a symlink to `/usr/bin` anyway.

Reply:
I think it is a bit safer to run it one by one file, in case the list of files becomes too big.

Reply:
It is also slower. But you are probably right that in case of really long list of arguments it could be problematic.